File: style-and-theme/LatihanStyleAndTheme/java/app/src/main/java/com/dicoding/picodiploma/myviewandviewgroup/MainActivity.java
package com.dicoding.picodiploma.myviewandviewgroup;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;

import com.dicoding.picodiploma.myviewandviews.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        if (getSupportActionBar() != null) {
            getSupportActionBar().setTitle("Google Pixel");
        }
    }
}

File: MyViewAndViews/app/src/main/java/com/dicoding/myviewandviews/MainActivity.kt
package com.dicoding.myviewandviews

import android.os.Bundle
import androidx.appcompat.app.ActionBar
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        if (supportActionBar != null) {
            (supportActionBar as ActionBar).title = "Google Pixel"
        }    
    }
}
